code/modules/mob/living/simple_animal/hostile/zombie.dm

```diff
@@ -51,6 +51,14 @@
 			visible_message("<span class='danger'>[src] tears [L] to pieces!</span>")
 			src << "<span class='userdanger'>You feast on [L], restoring your health!</span>"
 			revive(full_heal = 1)
+			
+	if(istype(target, /obj/machinery/door/airlock))
+		src << "<span class='notice'>You start tearing apart the airlock...</span>"
+		playsound(src.loc, 'sound/hallucinations/growl3.ogg', 50, 1)
+		if(do_after(src, 250, target))
+			playsound(src.loc, 'sound/hallucinations/far_noise.ogg', 50, 1)
+			qdel(target)
```

Comment:
before this line:
var/obj/structure/door_assembly/door = new(src.loc)
door.density = 0
door.anchored = 1

and this is optional:
door.name = "destroyed airlock"
door.desc = "A broken airlock. Looks like it wont be keeping much out now."